app/models/role.rb

```diff
@@ -37,6 +37,27 @@ def access_level
       return 1
     end
   end
+  # Returns a hash of hashes where each key represents an access level (e.g. see access_level method to understand the integers)
+  # This method becomes useful for generatic template messages (e.g. permissions change notification mailer)
+  def self.access_level_messages
+    {
+      3 => {
+        :type => _('co-owner'),
```

Comment:
looks very useful. Just switch over to the newer `type: _('co-owner'),` syntax instead of `:type => _('co-owner'),`

Reply:
The reason I'm using arrow hash is because I don't want to mix old/new syntax for that hash definition. It is needed to preserve the first level Integer keys because:

- access_level method defined above returns Integer and only would match for Integer keys
- and if switched to ':' syntax it raises an error since Integer keys are not coerced to symbols in the new syntax

In case you are not familiar with the new colon syntax, it only coerces keys to symbols for TrueClass, FalseClass and StringClass, the rest of types raises a syntax error. I personally discourage this new syntax unless you are aware that every key gets coerced to symbol, if possible